blocks/multi-panel-carousel/multi-panel-carousel.js

```diff
@@ -427,7 +473,7 @@ function addDiscoverLink(carouselBody, discoverLink) {
 }
 
 function getHighlightDiv(block) {
-  const predicationDiv = block.querySelectorAll(':scope > div')[2].children[1];
+  const predicationDiv = block.querySelectorAll(':scope > div')[3].children[1];
```

Comment:
This makes hard assumptions on a block to be authored in a particular way like "the 4th row in the block must be this or that" . Can we employ a method like `getBlockElements(key)` in which you give the `key` and get the corresponding markup div in response.

Reply:
@sdmcraft yes that has been discussed in the pr https://github.com/aemsites/icicidirect/pull/137.

Reply:
I'd suggest we have a simplified `getBlockElements` in #137 and merge it and then use that here.

Reply:
@sdmcraft updated pr #137 to have a simple blockMarkUp reader. Once that is merge, I will remove these changes from here

Reply:
PR #137 has been merged and update this pr basis on that